.travis.yml

```diff
@@ -1,15 +1,27 @@
 language: python
+group: travis_latest
 dist: xenial
 
+git:
+  depth: 3
+  quiet: true
+
 python:
-    - 3.5
-    - 3.6
-    - 3.7
+  - 3.8
+
```

Comment:
We should test against 3.5, 3.6, 3.7 as well, at least under linux. Is this still the case?

Reply:
Thanks for noting that! Now it should be fixed!